apps/studio/schemas/person.ts

```diff
@@ -19,9 +20,74 @@ export default defineType({
       type: "string",
     }),
     defineField({
-      name: "linkedinURL",
-      title: "LinkedIn URL",
-      type: "url",
+      name: "email",
+      title: "Email",
+      type: "email",
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: "socials",
+      title: "Socials",
+      type: "array",
+      of: [
+        {
+          type: "object",
+          fields: [
+            defineField({
+              name: "platform",
+              title: "Platform",
+              type: "string",
+              options: {
+                list: ["LinkedIn", "Personal Website"],
+              },
+              validation: (Rule) => Rule.required(),
+            }),
+            defineField({
+              name: "label",
+              title: "Label",
+              description:
+                "Primary identifier of platform (i.e. Username). Include a prefix character (@, ~, etc.) if used by brand.",
+              type: "string",
+            }),
+            defineField({
+              name: "link",
+              title: "Link",
+              type: "url",
+              validation: (Rule) =>
+                Rule.required().uri({ scheme: ["http", "https", "mailto"] }),
+            }),
+          ],
+          preview: {
+            select: {
+              title: "platform",
+              platform: "platform",
+              subtitle: "label",
+              link: "link",
+            },
+            prepare({ title, platform, subtitle, link }) {
+              let icon;
+              switch (platform.toLowerCase()) {
+                case "linkedin":
+                  icon = SiLinkedin;
+                  break;
+                case "personal website":
+                  icon = UserIcon;
```

Comment:
I think LinkIcon makes more sense here.
https://lucide.dev/icons/link

Reply:
Is this a separate icon library? We should probably stick to either Lucide or Simple Icons so there aren't any overlapping icons between the two.

Reply:
It's in Lucide, you can just change your import at the top to: `import { UserIcon, EarthGlobeIcon, LinkIcon } from "@sanity/icons";`

Reply:
Oof, another case of me not being able to find the icon. Fixed in [`09d1d86`](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/09d1d8696fd479b7a0ae77d1e6d1cd0219bddd72)